### DISCUSSION TITLE
Deepbench benchmark crashes

### DISCUSSION THREAD

**Discussion by gkarfakis19** [accel-sim/accel-sim-framework#261] [Q&A]
Hi. I am trying to use accel-sim with the gpgpu-apps repo. Whenever I run any of the deepbench benchmarks (tensor or not) included with the QV100 and Accelsim in SASS  mode, they immediately crash with a core dump. I am using CUDA 11.0.

Polybench and the rodinia benchmarks work as expected.

Does anybody know how to fix this? Thanks!

**Comment by JRPan** [comment]
Do you know the exact line causing the error? 

You can use gdb to run the program and see the call stack, or analyze the core dump. 

That would be very helpful and hopefully, we can narrow down the issue. 

Thanks
